pcsx2/gui/FrameForGS.cpp

```diff
@@ -183,14 +190,21 @@ void GSPanel::DoResize()
 	else if( arr > 1 )
 		viewport.y = (int)( (double)viewport.y/arr + 0.5);
 
+	if (g_Conf->GSWindow.ScalingType != ScalingType_Fit && g_Conf->GSWindow.ScalingType != ScalingType_Fill && acceptedImageSize) {
```

Comment:
I think it would be easier to avoid the negation
`..ST ==ScalingType_Integer || ..ST == ScalingType_Centered`